### ISSUE TITLE
`"enabled": false` triggers NPE in page assembler

### ISSUE THREAD

**Issue by x1B** [LaxarJS/laxar-tooling#26]
Looks like we need to do more (or earlier) filtering of disabled widget/composition items:

```js
Module build failed: TypeError: Cannot read property 'name' of undefined
    at laxar-tooling/src/page_assembler.js:402:28
    at Array.map (native)
    at itemName (laxar-tooling/src/page_assembler.js:402:11)
    at ensureItemHasId (laxar-tooling/src/page_assembler.js:413:25)
    at Array.forEach (native)
    at laxar-tooling/src/page_assembler.js:420:16
    at forEachArea (laxar-tooling/src/page_assembler.js:495:10)
    at generateMissingIds (laxar-tooling/src/page_assembler.js:419:7)
    at laxar-tooling/src/page_assembler.js:106:13
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by jpommerening** [comment]
I think I see what you mean with earlier … iterating over area items is all over the place …

**Comment by x1B** [comment]
@jpommerening haven't gotten around to this yet, feel free to re-assign if that makes sense to you :)

**Comment by x1B** [comment]
Fixed on `master` (`v2.0.0`).
